lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -86,19 +86,22 @@ sub rendered_markdown {
 package CommentsMarkdownParser;
 require Text::Markdown;
 our @ISA = qw/Text::Markdown/;
+use Regexp::Common qw/URI/;
```

Comment:
if you have a new direct dependency, please add it to cpanfile - preferably in the same commit introducing the dependency.